ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/type-component/array-type.component.ts

```diff
@@ -19,8 +19,8 @@ import { FieldArrayType } from '@ngx-formly/core';
 @Component({
   selector: 'gio-fjs-array-type',
   template: `
-    <div>
-      <legend *ngIf="to.label">{{ to.label }}</legend>
+    <div class="wrapper">
+      <h4 *ngIf="to.label">{{ to.label }}</h4>
```

Comment:
For accessibility reason, we should not use `h4` tags for styling only. Maybe we can replace this with 
```suggestion
      <div class="mat-subheading-1" *ngIf="to.label">{{ to.label }}</div>
```

Reply:
👌 
we just have to keep the margin underneath or I have to add it